utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/encode/JsonSchemaOutputValidator.scala

```diff
@@ -35,6 +35,12 @@ class JsonSchemaOutputValidator(validationMode: ValidationMode) extends LazyLogg
 
   private val valid = Validated.Valid(())
 
+  private val additionalTypesMapping: Map[Class[_], Set[Class[_]]] = Map(
+    classOf[java.lang.Long] -> Set(classOf[java.lang.Integer]),
+    classOf[java.lang.Float] -> Set(classOf[java.lang.Integer], classOf[java.lang.Long]),
+    classOf[java.lang.Double] -> Set(classOf[java.lang.Integer], classOf[java.lang.Long], classOf[java.lang.Float]),
+  )
+
```

Comment:
can you use https://github.com/TouK/nussknacker/pull/4065/files#diff-392c91bfe839de8a149d79b43340294b4faf257fb43b707f8b1c99e914f37c1fR179 this function after merge of this PR instead of this mapping?

Reply:
done